include/PlayStateAbstract.hpp

```diff
@@ -0,0 +1,35 @@
+//
+// Created by Jakub Kiermasz on 2019-05-25.
+//
+
+#ifndef HEXADOKU_PLAYSTATEABSTRACT_HPP
+#define HEXADOKU_PLAYSTATEABSTRACT_HPP
+
+#include "AbstractState.hpp"
+#include "Model/Board.hpp"
+#include <ctime>
+#include <memory>
+#include <iostream>
+
+class PlayStateAbstract : public AbstractState {
```

Comment:
Jeśli jest to Abstract, który nie powinien byc używany jako osobny stan a jedynie stany po nim dziedziczące,
należałoby dodać metodę czysto wirtualną. Jakiś destruktor albo HandleInput mogą być dobrymi kandydatami

Reply:
HandleInput już jest pure virtual, mój błąd